ironfish-cli/src/commands/chain/download.ts

```diff
@@ -40,11 +40,16 @@ export default class Download extends IronfishCommand {
       char: 'p',
       parse: (input: string) => Promise.resolve(input.trim()),
       required: false,
-      description: 'Path to snapshot file',
+      description: 'Path to a downloaded snapshot file to import',
     }),
     confirm: Flags.boolean({
       default: false,
-      description: 'Confirm without asking',
+      description: 'Confirm download without asking',
+    }),
+    cleanup: Flags.boolean({
+      default: true,
+      description: 'Remove downloaded snapshot file after import',
+      allowNo: true,
```

Comment:
should we add this as a hidden flag? Should most users need to use this?

Reply:
Setting to hidden -- most users shouldn't need to keep snapshots around after import